Tests/Core/AWSClientRuntimeTests/EventStream/AWSMessageEncoderStreamTests.swift

```diff
@@ -57,7 +57,7 @@ final class AWSMessageEncoderStreamTests: XCTestCase {
         let sut = EventStream.DefaultMessageEncoderStream(
             stream: baseStream,
             messageEncoder: messageEncoder,
-            marshalClosure: jsonMarshalClosure(requestEncoder: JSONEncoder()),
+            marshalClosure: TestEvent.marshal,
```

Comment:
JSON protocols now provide a code-generated unmarshal closure for streaming types.